src/main/groovy/com/blackducksoftware/integration/hub/api/HubMediaTypes.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Hub Common
+ *
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.api;
+
+public class HubMediaTypes {
+    public final static String VULNERABILITY_REQUEST_SERVICE = "application/vnd.blackducksoftware.list-1+json";
```

Comment:
I would add V1 into the name somewhere, since you'll likely need V2 in the near future.